src/main/java/wolox/training/controllers/UserController.java

```diff
@@ -198,4 +209,82 @@ public User deleteBookToUser(@ApiParam(value = "id to find the user") @PathVaria
         user.deleteBook(book);
         return userRepository.save(user);
     }
+
+    /**
+     * Method that allows updating the password of a specific user
+     *
+     * @param loginDTO: object containing password and username
+     * @param id: this is the unique identifier of the user
+     * @return {@link ResponseEntity}
+     */
+    @PatchMapping("/{id}/password")
+    @ApiOperation(value = "Given the id of the user, the password of the user will be updated, return the user", response = User.class)
+    @ApiResponses(value = {
+            @ApiResponse(code = 200, message = "Password user updated"),
+            @ApiResponse(code = 404, message = "User not found"),
+            @ApiResponse(code = 405, message = "Method Not Allowed"),
+            @ApiResponse(code = 401, message = "Access unauthorized."),
+            @ApiResponse(code = 403, message = "Access unauthorized."),
+            @ApiResponse(code = 500, message = "Internal Server Error")
+    })
+    @ResponseStatus(HttpStatus.OK)
+    public ResponseEntity<Void> updatePassword(@RequestBody LoginDTO loginDTO, @PathVariable Long id) {
```

Comment:
Remember to change the method that updates the user so that the password can't be updated with that method (set the password to the current password) 🙂